src/App.tsx

```diff
@@ -1,15 +1,40 @@
-import React from 'react';
+import React, { useEffect, useState } from "react";
 
-import { TextSection } from './TextSection';
+import { TextSection } from "./Components/TextSection";
+import Card from "./Components/Card";
 
 function App() {
-    return (
-        <React.Fragment>
-            <TextSection id="welcome" />
-            <TextSection id="blog_post" />
-            <TextSection id="kitchen_sink" />
-        </React.Fragment>
-    );
+  const [slideToShow, setSlideToShow] = useState(1);
+
+  useEffect(() => {
+    setTimeout(() => {
+      if (slideToShow < 3) {
+        setSlideToShow(slideToShow + 1);
+      } else {
```

Comment:
Due to the allotted time, I was not able to experiment with emotion's annotation but I did include the library and added basic styling to show that I could use the library.